airflow/plugins_manager.py

```diff
@@ -97,6 +99,7 @@ def validate(cls):
         except Exception as e:
             log.exception(e)
             log.error('Failed to import plugin %s', filepath)
+            import_errors[filepath] = str(e)
```

Comment:
Please make sure you also clean import_errors when the a particular plugin is fixed.

Reply:
I believe the only way to get the Airflow UI to reload a broken plugin is to restart the webserver.

Reply:
Yes, this is true